src/OrbitGl/TriangleToggle.cpp

```diff
@@ -17,13 +17,15 @@
 #include "Track.h"
 
 TriangleToggle::TriangleToggle(State initial_state, StateChangeHandler handler,
-                               TimeGraph* time_graph, TimeGraphLayout* layout, Track* track)
+                               TimeGraph* time_graph, TimeGraphLayout* layout, Track* track,
+                               uint32_t indentation_level)
```

Comment:
Should the `TriangleToggle` class "know" about indentation level? This seems like an unnecessary coupling of meaning. Just pass the size here?

Reply:
Good idea. Done.